test/test_generate_key.sh

```diff
@@ -18,7 +18,7 @@ if [[ ! "$RESULT" =~ $EXPECTED ]]; then
     exit 1
 fi
 
-RESULT2=$(./bin/near generate-key generate-key-test --networkId default)
+RESULT2=$(./bin/near generate-key generate-key-test --networkId ci)
```

Comment:
Same question about all these changes from `default` to `ci` – should they be in a separate PR? I don't see any description about these.

Reply:
without a modification like this tests break. I don't understand how they haven't been breaking before this. I pushed a change adding an environment variable instead.
it sounds like you're confused and expected more info. how should i communicate this? commit is too short to explain. should I be updating the PR description with each change? At this point I'm firefighting the broken tests. I can add that

Reply:
Ideally I would like to see one or two separate PRs laser-focused on fixing these issues, as it leaves more of a paper-trail and makes space for better discussion. But whether a totally new PR is necessary is a judgement call, and I'm ok with fitting these small fixes into this one as separate commits, each with a good description.